.JamesLau-fe-project/src/Components/Comments.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+import { getCommentsByArticleId } from "../api";
+import CommentCard from "./CommentCard";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((data) => {
+      setComments(data);
+      setIsLoading(false);
+    });
+  }, [article_id]);
```

Comment:
Is this dependancy required? The Comments component is re-rendered if the ArticlePage changes and so the use effect will run again anyway